projects/netgrif-components/src/lib/data-fields/i18n-field/i18n-text-field/i18n-text-field.component.scss

```
@@ -11,7 +11,7 @@
     font-size: 14px;
     border: 0;
     margin-left: 8px;
-    color: #64748B;
+    //color: #64748B;
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)